<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1131</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(129, 194, 100);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Employees</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/Screenshot_2024-09-22_143229-removebg-preview.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:700; color:#d2e7ff;&quot;&gt;PharmaFlow&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>981</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 227, 251);</string>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>271</width>
         <height>371</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);

</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom Produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID Produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Categorie</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date Exp</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix Achat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix vente</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>25</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_expiration">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idp">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>50</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_quantite">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_categorie">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Antalgiques</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Antibiotiques</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Digestif</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Respiratoire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vitamines et suppl√©ments</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cosmetiques</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Anti-inflammatoires</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paracetamol</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prixachat">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prixvente">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>170</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>200</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>240</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_afficher">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>280</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>320</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="tableView_stock">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>621</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Fournisseurs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Stock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Transactions</string>
      </property>
     </widget>
     <zorder>pushButton_6</zorder>
     <zorder>pushButton_7</zorder>
     <zorder>label_10</zorder>
     <zorder>widget</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>label_9</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>841</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(210, 231, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_recherche">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>RECHERCHE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_trier">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>550</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TRIER</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_recherche">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>recherche_par_id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>recherche_par_nom</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_Trier">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>590</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>trier_par_date_expiration</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>trier_par_quantite</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_exportpdf">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>540</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>exporter Pdf</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/Screenshot_2024-09-22_143229-removebg-preview.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Employees</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Fournisseurs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Transactions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Stock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stats">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>110</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <zorder>tableView_2</zorder>
     <zorder>pushButton_recherche</zorder>
     <zorder>pushButton_trier</zorder>
     <zorder>comboBox_recherche</zorder>
     <zorder>comboBox_Trier</zorder>
     <zorder>pushButton_exportpdf</zorder>
     <zorder>label_21</zorder>
     <zorder>lineEdit_recherche</zorder>
     <zorder>pushButton_8</zorder>
     <zorder>pushButton_9</zorder>
     <zorder>pushButton_10</zorder>
     <zorder>pushButton_11</zorder>
     <zorder>pushButton_12</zorder>
     <zorder>pushButton_stats</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
